pennylane/measure.py

```diff
@@ -346,14 +354,13 @@ def circuit(x):
            [1, 1],
            [0, 0]])
 
+    .. note::
 
-    Args:
-        op (Observable or None): a quantum observable object
-        wires (Sequence[int] or int or None): the wires we wish to sample from, ONLY set wires if op is None
-
-    Raises:
-        QuantumFunctionError: `op` is not an instance of :class:`~.Observable`
-        ValueError: Cannot set wires if an observable is provided
+        QNodes that return samples cannot, in general, be differentiated, since the derivative
+        with respect to a sample --- a stochastic process --- is ill-defined. The one exception
+        is if the QNode uses the parameter-shift method (``diff_method="parameter-shift"``), in which
+        case ``qml.sample(obs)`` is interpreted as a single-shot expectation value of the
+        observable ``obs``.
```

Comment:
@mariaschuld I added this here, let me know what you think!

Reply:
Nice, I think this is clear!